src/__tests__/to-be-visible.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Modal, View } from 'react-native';
+import { render } from '@testing-library/react-native';
+
+describe('.toBeVisible', () => {
+  test.each([
```

Comment:
Github/link seems to be complaining about missing `key` prop, Could you refactor these tests to be regular. not `.each` tests for readability.

Reply:
I fixed this issue when migrating the test file to TypeScript in [this commit](https://github.com/testing-library/jest-native/pull/73/commits/861cc729f7857aed8d05f120e10aa293b4829da0).